.github/workflows/braintrust-evals.yml

```
@@ -88,6 +87,8 @@ jobs:
 
       - name: Run Evals
         uses: braintrustdata/eval-action@c0dd75b29984a0cc63a827d6e8da2f23f2752be4 # v1.0.16
+        permissions:
```

actionlint (1.7.12)

[error] unexpected key "permissions" for step to execute action. expected one of "continue-on-error", "env", "id", "if", "name", "timeout-minutes", "uses", "with"

(syntax-check)